tasks/main.yml

```diff
@@ -477,6 +477,26 @@
   when:
     - mssql_tls_enable is not none
   block:
+    - block:
+        - name: Generate certificates using the certificate role
+          include_role:
+            name: fedora.linux_system_roles.certificate
+          vars:
+            certificate_requests: |
+              {% for _cert in mssql_certificates %}
+              {%   set _ = _cert.__setitem__('owner', 'mssql') %}
+              {%   set _ = _cert.__setitem__('group', 'mssql') %}
+              {%   set _ = _cert.__setitem__('mode', '0600') %}
+              {% endfor %}
+              {{ mssql_certificates }}
+
+        - name: Generate certificates using the certificate role
+          set_fact:
+            mssql_tls_cert: "{{ mssql_certificates.0.name }}.crt"
+            mssql_tls_private_key: "{{ mssql_certificates.0.name }}.key"
+
+      when: mssql_certificates | length > 0
```

Comment:
Someone said that the latest version of ansible-lint was complaining about putting `when` in blocks underneath tasks. I think it is more readable when `when` is set after the `block` statement.